File: list-App/TodoApp.py
tasks = []

while True:
    command = input("Commands: add <task>, list, delete <index>, exit\n")

    if command.startswith("add "):
        task = command[4:]
        tasks.append(task)
        print(f"Added task: {task}")
    elif command == "list":
        print("To-Do List:")
        for i, task in enumerate(tasks):
            print(f"{i}: {task}")
    elif command.startswith("delete "):
        index = int(command.split()[1])
        if 0 <= index < len(tasks):
            print(f"Removed task: {tasks.pop(index)}")
        else:
            print("Task not found.")
    elif command == "exit":
        break
    else:
        print("Invalid command.")

File: list-App/TodoApp.java
import java.util.ArrayList;
import java.util.Scanner;

public class TodoApp {
    private static ArrayList<String> tasks = new ArrayList<>();

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String command;

        while (true) {
            System.out.println("Commands: add <task>, list, delete <index>, exit");
            command = scanner.nextLine();

            if (command.startsWith("add ")) {
                String task = command.substring(4);
                tasks.add(task);
                System.out.println("Added task: " + task);
            } else if (command.equals("list")) {
                listTasks();
            } else if (command.startsWith("delete ")) {
                int index = Integer.parseInt(command.split(" ")[1]);
                if (index >= 0 && index < tasks.size()) {
                    System.out.println("Removed task: " + tasks.get(index));
                    tasks.remove(index);
                } else {
                    System.out.println("Task not found.");
                }
            } else if (command.equals("exit")) {
                break;
            } else {
                System.out.println("Invalid command.");
            }
        }
        scanner.close();
    }

    private static void listTasks() {
        System.out.println("To-Do List:");
        for (int i = 0; i < tasks.size(); i++) {
            System.out.println(i + ": " + tasks.get(i));
        }
    }
}
